### DISCUSSION TITLE
MAC channel access failure/MAC no ack - if you have a SVS SoundPath Wireless audio adapter

### DISCUSSION THREAD

**Discussion by KDanielK** [Koenkk/zigbee2mqtt#18664] [Devices]
I had a lot of problems with my network and I tried to change channels, location, length of USB cable, firmware etc. After a while it struck me that I was using a SVS SoundPath Wireless Audio Adapter that operates on the 2G band
https://www.svsound.com/products/soundpath-wireless-audio-adapter

When I disconnected this device my network started working and I no longer have any errors. I bought the newer version from SVS called Tri-band and everything still works 100% https://www.svsound.com/products/soundpath-tri-band-wireless-audio-adapter

If you are experiencing issues with your network and have a SVS SoundPath wireless adapter please consider disconnecting it for a while.

Perhaps this should be added to the network guide? https://www.zigbee2mqtt.io/advanced/zigbee/02_improve_network_range_and_stability.html#try-different-orientations-of-the-adapter

**Comment by Koenkk** [comment]
You can *Help to make the docu better* in the bottom section of the page to make a pr
